hotham/src/contexts/physics_context.rs

```diff
@@ -20,9 +19,9 @@ pub struct PhysicsContext {
     pub narrow_phase: NarrowPhase,
     pub rigid_bodies: RigidBodySet,
     pub island_manager: IslandManager,
-    pub collision_recv: Receiver<CollisionEvent>,
-    pub contact_force_recv: Receiver<ContactForceEvent>,
-    pub event_handler: ChannelEventCollector,
+    // pub collision_recv: Receiver<CollisionEvent>,
+    // pub contact_force_recv: Receiver<ContactForceEvent>,
+    // pub event_handler: ChannelEventCollector,
```

Comment:
Why is this removed?

Reply:
Good question!

- We never actually use Rapier events. We probably should, at some point in the future, because they're more efficient
- By removing them we can avoid an unnecessary crossbeam dependency